packages/sync/src/Queue.php

```diff
@@ -418,6 +427,29 @@ private function fetch_items( $limit = null ) {
 		return $items;
 	}
 
+	private function fetch_items_by_id( $items_ids ) {
+		global $wpdb;
+
+		$items_ids = implode(
```

Comment:
I know this is not a new pattern here, but in the spirit of not introducing not-recommended patterns, could we use `$wpdb->prepare()` instead? See this example of preparing for `IN()` clauses: https://developer.wordpress.org/reference/classes/wpdb/prepare/#comment-1557

Reply:
https://github.com/Automattic/jetpack/pull/13124/commits/8c6795dbd179faaa524b115113d1850311175b86